src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -139,53 +201,122 @@ public void reportIncludesItemsInTransitToDifferentServicePoints()
+    final LocalDate requestExpirationDate2 = new LocalDate(2020, 1, 12);
+    final String servicePointName1 = "Circ Desk 1";
+    final String servicePointName2 = "Circ Desk 2";
+    final String requestPatronGroup1 = "Jones, Steven";
+    final String requestPatronGroup2 = "Stuart, Rebecca";
+
     loansFixture.checkOutByBarcode(smallAngryPlanet);
     loansFixture.checkOutByBarcode(nod);
 
-    createRequest(smallAngryPlanet, steve, firstServicePointId);
-    createRequest(nod, rebecca, secondServicePointId);
+    createRequest(smallAngryPlanet, steve, firstServicePointId, requestDate1, requestExpirationDate1);
+    createRequest(nod, rebecca, secondServicePointId, requestDate2, requestExpirationDate2);
 
     loansFixture.checkInByBarcode(new CheckInByBarcodeRequestBuilder()
       .forItem(smallAngryPlanet)
-      .on(DateTime.now(DateTimeZone.UTC))
+      .on(checkInDate1)
       .at(secondServicePointId));
-    loansFixture.checkInByBarcode(nod);
+    loansFixture.checkInByBarcode(new CheckInByBarcodeRequestBuilder()
+      .forItem(nod)
+      .on(checkInDate2)
+      .at(firstServicePointId));
 
     List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
 
     assertThat(items.size(), is(2));
+    JsonObject firstItemJson = getRecordById(items, smallAngryPlanet.getId()).get();
+    verifyItem(firstItemJson, smallAngryPlanet, firstServicePointId);
+    verifyLocation(firstItemJson);
+    verifyRequest(firstItemJson, requestDate1, requestExpirationDate1, requestPatronGroup1, servicePointName1);
+    verifyLoan(firstItemJson, checkInDate1, servicePointName2,
+      "cd2", "Circulation Desk -- Back Entrance");
+
+    JsonObject secondItemJson = getRecordById(items, nod.getId()).get();
+    verifyItem(secondItemJson, nod, secondServicePointId);
+    verifyLocation(secondItemJson);
+    verifyRequest(secondItemJson, requestDate2, requestExpirationDate2, requestPatronGroup2, servicePointName2);
+    verifyLoan(secondItemJson, checkInDate2);
   }
 
-  private void createRequest(InventoryItemResource smallAngryPlanet, IndividualResource steve, UUID secondServicePointId)
+  private void createRequest(InventoryItemResource smallAngryPlanet,
+                             IndividualResource steve, UUID secondServicePointId,
+                             DateTime requestDate, LocalDate requestExpirationDate)
     throws InterruptedException,
     MalformedURLException,
     TimeoutException,
     ExecutionException {
+    RequestBuilder.Tags tags = new RequestBuilder.Tags(Arrays.asList("tag1", "tag2"));
     RequestBuilder secondRequestBuilderOnItem = new RequestBuilder()
       .open()
       .hold()
       .withPickupServicePointId(secondServicePointId)
       .forItem(smallAngryPlanet)
+      .withTags(tags)
+      .withRequestDate(requestDate)
+      .withRequestExpiration(requestExpirationDate)
       .by(steve);
+
     requestsClient.create(secondRequestBuilderOnItem);
   }
 
```

Comment:
Could you add tests for:
1. Items in transit with empty request queue
2. Items in transit with more than one open request in a queue (check for first request)

Reply:
Done